cli/cmdlineparser.cpp

```diff
@@ -421,11 +421,18 @@ CmdLineParser::Result CmdLineParser::parseFromArgs(int argc, const char* const a
             return Result::Exit;
         }
 
+        if (std::strcmp(argv[i], "--filesdir") == 0) {
+#ifdef FILESDIR
+            mLogger.printRaw(FILESDIR); // TODO: should not include newline
```

Comment:
This should not include the newline because that means you can simply assign this to a variable but you always have to strip the newline which is extremely annoying in a shell environment.

Reply:
It makes sense to me that the newline is added. But maybe the name can be clarified instead.